src/bridge/MatrixHandler.ts

```diff
@@ -30,6 +31,12 @@ const MSG_PMS_DISABLED_FEDERATION = "[Bridge] Sorry, PMs are disabled on this br
 const DEFAULT_EVENT_CACHE_SIZE = 4096;
 /* Length of the source text in a formatted reply message */
 const REPLY_SOURCE_MAX_LENGTH = 32;
+// How many seconds needs to pass between a message and a reply to it to switch to the long reply format
+const DEFAULT_SHORT_REPLY_TRESHOLD_SECONDS = 5 * 60;
```

Comment:
We tend to store values in ms across the bridge. In this case, I guess it's fine because the config is in seconds?

Reply:
Correct.

The config file uses a plethora of units, as in `floodDelayMs`, `quitsPerSecond` and even `idleForHours`. Interestingly, `ircClients.idleTimeout` is the only one that doesn't have its unit in the name, and its unit turns out to be seconds also :) It'd be hellish to fix it now, but it is at least nicely documented to be that way.

Reply:
Fair, one day we should refactor the config to be sane in many ways, and just deprecate the old ways.